### ISSUE TITLE
lsass.exe crash after updating Sandboxie Plus

### ISSUE THREAD

**Issue by casualuser2010** [sandboxie-plus/Sandboxie#4346]
### Describe what you noticed and did

I downloaded [Sandboxie-Plus-x86-v1.14.10.exe](https://github.com/sandboxie-plus/Sandboxie/releases/download/v1.14.10/Sandboxie-Plus-x86-v1.14.10.exe) on a Windows 7 Starter. After installing I got a window pop-out saying "Windows has encountered a critical error and will restart automatically in one minute. Please save all work now.". After a minute my system restarted without showing "shuttin down" message. When I went to Event Viewer I saw error message relating with "lsass.exe crashing" (since my Windows install is in spanish I decided to keep it as is to avoid confusion):

```
Event ID 1000: Application Error
Nombre de la aplicación con errores: lsass.exe, versión: 6.1.7601.18933, marca de tiempo: 0x55a68b8d
Nombre del módulo con errores: ntdll.dll, versión: 6.1.7601.18933, marca de tiempo: 0x55a69d9f
Código de excepción: 0xc0000005
Desplazamiento de errores: 0x000576e4
Id. del proceso con errores: 0x27c
Hora de inicio de la aplicación con errores: 0x01db2ad01e294abf
Ruta de acceso de la aplicación con errores: C:\Windows\system32\lsass.exe
Ruta de acceso del módulo con errores: C:\Windows\SYSTEM32\ntdll.dll
Id. del informe: 8f86defd-96c8-11ef-96b8-00e048003b9c

Event ID 1001: Windows Error Reporting
Depósito con errores , tipo 0
Nombre de evento: APPCRASH
Respuesta: No disponible
Id. de archivo CAB: 0

Firma del problema:
P1: lsass.exe
P2: 6.1.7601.18933
P3: 55a68b8d
P4: ntdll.dll
P5: 6.1.7601.18933
P6: 55a69d9f
P7: c0000005
P8: 000576e4
P9: 
P10: 

Archivos adjuntos:

Es posible que estos archivos estén disponibles aquí:
C:\ProgramData\Microsoft\Windows\WER\ReportQueue\AppCrash_lsass.exe_2c27946f3efe6e2f06dc8a592d8c7b0b1a721f9_15026b8e

Símbolo de análisis: 
Nueva búsqueda de una solución: 0
Id. de informe: 8f86defd-96c8-11ef-96b8-00e048003b9c

Event ID 1015: Wininit
Error en el proceso crítico del sistema C:\Windows\system32\lsass.exe con código de estado 255. El equipo debe reiniciarse ahora.
```

### How often did you encounter it so far?

Each time I update my Sandboxie client

### Expected behavior

I shouldn't see any kind of Windows error.

### Affected program

Not relevant

### Download link

Not relevant

### Where is the program located?

Not relevant to my request.

### Did the program or any related process close unexpectedly?

No, not at all.

### What version of Sandboxie are you running now?

Sandboxie-Plus-x86-v1.14.10

### Is it a new installation of Sandboxie?

I just updated Sandboxie from a previous version (I remember which one it is).

### In which sandbox type you have this problem?

Not relevant to my request.

### Can you reproduce this problem on a new empty sandbox?

Not relevant to my request.

### What is your Windows edition and version?

Windows 7 Starter SP1 (compilation 7601)

### In which Windows account you have this problem?

A local account (Administrator).

### Please mention any installed security software

Avast! Antivirus Free

**Comment by bastik-1001** [comment]
I had this happen to me once, on my Windows 7 machine, after I returned from hibernation (deeper standby) and concluded that something had been corrupted either as it was written to disk or back to memory, I did not consider Sandboxie to be causing this.

Now, it happened yesterday, after updating Firefox and Sandboxie (to Plus 1.15.1), but I had not looked into it.

According to the system log, this happened thrice.

2024-09-07:
> Name der fehlerhaften Anwendung: lsass.exe, Version: 6.1.7601.24545, Zeitstempel: 0x5e0ead35
Name des fehlerhaften Moduls: ntdll.dll, Version: 6.1.7601.24545, Zeitstempel: 0x5e0eb67f
Ausnahmecode: 0xc0000005
Fehleroffset: 0x00000000000323f0
ID des fehlerhaften Prozesses: 0x2d8
Startzeit der fehlerhaften Anwendung: 0x01dafce14785f1c8
Pfad der fehlerhaften Anwendung: C:\Windows\system32\lsass.exe
Pfad des fehlerhaften Moduls: C:\Windows\SYSTEM32\ntdll.dll
Berichtskennung: f4cec16d-6cd2-11ef-b64e-00241dd70bdd

2024-09-19:
> Name der fehlerhaften Anwendung: lsass.exe, Version: 6.1.7601.24545, Zeitstempel: 0x5e0ead35
Name des fehlerhaften Moduls: ntdll.dll, Version: 6.1.7601.24545, Zeitstempel: 0x5e0eb67f
Ausnahmecode: 0xc0000005
Fehleroffset: 0x00000000000323f0
ID des fehlerhaften Prozesses: 0x2d8
Startzeit der fehlerhaften Anwendung: 0x01db00e02437119b
Pfad der fehlerhaften Anwendung: C:\Windows\system32\lsass.exe
Pfad des fehlerhaften Moduls: C:\Windows\SYSTEM32\ntdll.dll
Berichtskennung: 689402d6-76ad-11ef-b63a-00241dd70bdd

2024-10-29:
> Name der fehlerhaften Anwendung: lsass.exe, Version: 6.1.7601.24545, Zeitstempel: 0x5e0ead35
Name des fehlerhaften Moduls: ntdll.dll, Version: 6.1.7601.24545, Zeitstempel: 0x5e0eb67f
Ausnahmecode: 0xc0000005
Fehleroffset: 0x00000000000323f0
ID des fehlerhaften Prozesses: 0x2dc
Startzeit der fehlerhaften Anwendung: 0x01db26e2d70b1ff0
Pfad der fehlerhaften Anwendung: C:\Windows\system32\lsass.exe
Pfad des fehlerhaften Moduls: C:\Windows\SYSTEM32\ntdll.dll
Berichtskennung: b3d2507b-961d-11ef-bf1d-00241dd70bdd
